docs/how-to-docs/cert-manager-upgrade-from-psg.md

```diff
@@ -28,11 +28,13 @@ There are 2 possible approaches for the migration of `Ingress` resources. The hi
 
 For a more detailed understanding of how the manifest files need to be updated, please refer to section [Updating cert-manager resources for v0.13.1](#updating-cert-manager-resources-for-v0131) below.
 
-Option 1 below is strongly recommended as the approach.
+Option 1 is the simplest and is appropriate if you can afford some down-time.
```

Comment:
`some down-time` - suggest what that is, e.g. are we talking seconds, minutes, hours potentially?

Reply:
updated; added:
 (typically a couple of minutes, but longer if there is an issue